resources/functionMap_php80delta.php

```diff
@@ -41,6 +41,8 @@
 		'date_timezone_set' => ['DateTime', 'object'=>'DateTime', 'timezone'=>'DateTimeZone'],
 		'explode' => ['list<string>', 'separator'=>'non-empty-string', 'str'=>'string', 'limit='=>'int'],
 		'fdiv' => ['float', 'dividend'=>'float', 'divisor'=>'float'],
+		'filter_input' => ['mixed', 'type'=>'INPUT_GET|INPUT_POST|INPUT_COOKIE|INPUT_SERVER|INPUT_ENV', 'variable_name'=>'string', 'filter='=>'int', 'options='=>'array|int'],
```

Comment:
Please do stricter parameter types in `functionMap_bleedingEdge.php`. If this is only PHP 8+ specific, please create `functionMap_php80delta_bleedingEdge.php`. Thank you!